ui/views.py

```diff
@@ -227,6 +227,10 @@ def dispatch(self, *args, **kwargs):
     def extra_context(self):
         """Add to the context."""
         context = super(RepositoryView, self).extra_context()
+        vocabularies = [
+            x.replace(" ", "_")
```

Comment:
I understand, but it's hard(er) to keep track of the logical connection with just the `replace`